cli/src/tests.rs

```diff
@@ -267,6 +267,33 @@ async fn cancel_invitation(tmp_path: TmpPath) {
         ));
 }
 
+#[rstest::rstest]
+#[tokio::test]
+async fn export_recovery_device(tmp_path: TmpPath) {
```

Comment:
Is it possible to check if `output` is populated with the expected data ?

Reply:
Sure